frontend/public/components/dashboard/project-dashboard/inventory-card.tsx

```diff
@@ -58,60 +52,42 @@ const ProjectInventoryItem = withDashboardResources(
     model,
     mapper,
     additionalResources,
-    additionalDynamicResources,
   }: ProjectInventoryItemProps) => {
     React.useEffect(() => {
       if (projectName) {
         const resource = createFirehoseResource(model, projectName);
         watchK8sResource(resource);
-        if (additionalResources) {
-          additionalResources.forEach((r) => watchK8sResource({ ...r, namespace: projectName }));
-        }
         return () => {
           stopWatchK8sResource(resource);
-          if (additionalResources) {
-            additionalResources.forEach(stopWatchK8sResource);
-          }
         };
       }
-    }, [watchK8sResource, stopWatchK8sResource, projectName, model, additionalResources]);
+    }, [watchK8sResource, stopWatchK8sResource, projectName, model]);
 
     const resourceData = _.get(resources.resource, 'data', []) as FirehoseResult['data'];
     const resourceLoaded = _.get(resources.resource, 'loaded');
     const resourceLoadError = _.get(resources.resource, 'loadError');
 
-    const additionalResourcesData = additionalResources
-      ? additionalResources.reduce((acc, r) => {
-          acc[r.prop] = _.get(resources[r.prop], 'data');
-          return acc;
-        }, {})
-      : {};
-    const additionalResourcesLoaded = additionalResources
-      ? additionalResources
-          .filter((r) => !r.optional)
-          .every((r) => _.get(resources[r.prop], 'loaded'))
-      : true;
-    const additionalResourcesLoadError = additionalResources
-      ? additionalResources
-          .filter((r) => !r.optional)
-          .some((r) => !!_.get(resources[r.prop], 'loadError'))
-      : false;
-
-    const dynamicResources = useK8sWatchResources(additionalDynamicResources || {});
-    const dynamicResourcesError = Object.values(dynamicResources).find((r) => r.loadError)
-      ?.loadError;
-    const dynamicResourcesLoaded = Object.keys(dynamicResources).every(
-      (key) => dynamicResources[key].loaded,
-    );
+    const additionalResourcesData = {};
+    let additionalResourcesLoaded = true;
+    let additionalResourcesLoadError = false;
+    if (additionalResources) {
```

Comment:
FYI - this code was taken from `public/components/dashboard/dashboards-page/cluster-dashboard/inventory-card.tsx` - `ClusterInventoryItem` component.